ci/run-benchmark.yaml

```diff
@@ -0,0 +1,55 @@
+platform: linux
+image_resource:
+  type: docker-image
+  source:
+    repository: theorf/google-cloud-sdk-helm
+inputs:
+  - name: eq-survey-runner-benchmark
+params:
+  IMPORT_EXISTING_PROJECT: true
```

Comment:
We should consider removing the `IMPORT_EXISTING_PROJECT` functionality given that it's the default in the pipelines.